GitOut/Features/Git/Stage/GitStageViewModel.cs

```diff
@@ -362,7 +362,18 @@ private async Task ResetSelectionAsync(IHunkLineVisitorProvider viewer)
                 );
                 string filename = Path.GetFileName(selectedChange.Path);
                 await Repository.ExecuteApplyAsync(patch);
-                snack.ShowSuccess($"Changes reset in {filename}", TimeSpan.FromSeconds(5), "UNDO", async () => await UndoPatchAsync());
+                _ = snack.ShowAsync(Snack.Builder()
+                    .WithMessage($"Changes reset in {filename}")
+                    .WithDuration(TimeSpan.FromSeconds(5))
+                    .AddAction("UNDO"))
```

Comment:
Not much for magic strings.. 🤔 can we move this to a constant, or perhaps convert it to enum values?